next/app/components/layouts/Header/Header.module.css

```
@@ -0,0 +1,23 @@
+.header {
+  display: flex;
+  background-color: #edc3e7;
+  font-size: 36px;
+  font-family: 'Montserrat';
+  font-weight: bold;
+  transform: scale(1.1, 1);
+  transform-origin: top left;
+  letter-spacing: -1.5px;
+  color: #2c3844;
+  padding: 20px;
+}
+
+.icon {
+  display: flex;
+  color: #2c3844;
+  padding-bottom: 3px;
+}
+
+.text {
+  padding-left: 25px;
```

基本は、要素の外側の余白にはmargin、要素の内側の余白にはpaddingを使いましょう！
そして、親要素と子要素の間（親子間）の余白はpadding、隣り合った要素（兄弟間）の余白はmarginを使うようにすると、迷うことがなくなりますよ！
```suggestion
  margin-left: 25px;
```